### ISSUE TITLE
Select/Selected button on record view should have hover state...

### ISSUE THREAD

**Issue by jkeck** [sul-dlss/SearchWorks#595]
...consistent with other buttons.  re: #561 

```
Select should technically have get a hover state to match the other controls, but it seems really hard to do. :-(
```
